### PR TITLE
Added getCSSDependencies to parallel getJSDependencies

### PR DESCRIPTION
**Pull request by driusan**
This makes it easier for modules to add custom CSS.

This should just rework how the code works, not what it does and should be backwards compatible.

### CODE REVIEW

**Comment by driusan** [comment]
Alex, can you test this? I think you were the latest person to ask for this feature.

**Comment by alisterdev** [comment]
@driusan will do

**Comment by alisterdev** [comment]
Works both in Loris modules and in module overrides inside the project 👍

File: php/libraries/NDB_Page.class.inc [hunk 1]
```diff
@@ -586,5 +586,44 @@ class NDB_Page
...
+     * interdependent files can be included in the correct order.
+     *
+     * @return array of strings with URLs to retrieve CSS resources
+     */
+    function getCSSDependencies()
+    {
+        $factory = NDB_Factory::singleton();
+        $config  = $factory->config();
+
+        $www     = $config->getSetting('www');
```
Comment (alisterdev) [inline php/libraries/NDB_Page.class.inc]: is there a difference of using that approach instead of 
```$baseurl = $factory->settings()->getBaseURL();```

Comment (driusan) [inline php/libraries/NDB_Page.class.inc]: I had just based it on the getJSDependencies() code directly above.. but, in fact, you're right: it should be using $factory->settings()->getBaseURL();

Comment (alisterdev) [inline php/libraries/NDB_Page.class.inc]: I found this approach in modules that override getJSDependencies(). Maybe its worth to update parent classes for consistency.

File: php/libraries/NDB_Page.class.inc [hunk 2]
```diff
@@ -586,5 +586,44 @@ class NDB_Page
...
+                 );
+
+        if (file_exists($paths['base'] . "modules/$TestName/css/$TestName.css")) {
+            error_log(
+                "WARNING: Relying on $TestName.css getting automatically loaded will"
+                . " soon be removed. "
+                . "Override getCSSDependencies() instead"
+            );
+
+            $files[] = $baseurl . "/$TestName/js/$TestName.css";
```
Comment (alisterdev) [inline php/libraries/NDB_Page.class.inc]: Just noticed that this line references js folder instead of css. Can you make a quick PR to fix that?

**Pull request merged**